spec/aruba/api/core_spec.rb

```diff
@@ -0,0 +1,138 @@
+require 'spec_helper'
+require 'aruba/api'
+require 'fileutils'
+
+describe Aruba::Api::Core do
```

Comment:
Can we use `RSpec.describe`?

```suggestion
RSpec.describe Aruba::Api::Core do
```